packages/test-app/tests/integration/components/fade-text-test.js

```diff
@@ -54,7 +54,13 @@ module('Integration | Component | fade-text', function (hooks) {
   test('it fades tall text given as component argument', async function (assert) {
     this.set('longHtml', this.longHtml);
 
-    await render(hbs`<FadeText @text={{this.longHtml}} />`);
+    this.set('expanded', false);
+    this.set('onExpandAll', (isExpanded) => {
+      this.set('expanded', isExpanded);
```

Comment:
An assertion here that the method is called as often as expected (counter or something else) would be welcome.

Reply:
Do you mean like an `assert.expect()`?

Reply:
I interpreted this as putting an `assert.strictEqual(this.expanded, isExpanded)` in that `onExpandAll` definition, which then caused me to need `assert.expect()` calls :D